torchhub load model python version restriction

## ❔Question
Hi, it looks similar to the model loading opencv-python problem, could you help cancel the pytorchhub python version check? 
Also, can I ask what's the right way to load the model locally, thanks a lot!

## Additional context
Traceback (most recent call last):
  File "loitering_main.py", line 373, in <module>
    main()
  File "loitering_main.py", line 120, in main
    detector = torch.hub.load('ultralytics/yolov5', 'yolov5s',force_reload= True)
  File "/home/psl/.local/lib/python3.6/site-packages/torch/hub.py", line 370, in load
    model = _load_local(repo_or_dir, model, *args, **kwargs)
  File "/home/psl/.local/lib/python3.6/site-packages/torch/hub.py", line 399, in _load_local
    model = entry(*args, **kwargs)
  File "/home/psl/.cache/torch/hub/ultralytics_yolov5_master/hubconf.py", line 70, in yolov5s
    return _create('yolov5s', pretrained, channels, classes, autoshape, verbose, device)
  File "/home/psl/.cache/torch/hub/ultralytics_yolov5_master/hubconf.py", line 34, in _create
    exclude=('tensorboard', 'pycocotools', 'thop', 'opencv-python'))
  File "/home/psl/.cache/torch/hub/ultralytics_yolov5_master/utils/general.py", line 151, in check_requirements
    check_python()  # check python version
  File "/home/psl/.cache/torch/hub/ultralytics_yolov5_master/utils/general.py", line 144, in check_python
    assert result, f'Python {minimum} required by YOLOv5, but Python {current} is currently installed'
AssertionError: Python 3.7.0 required by YOLOv5, but Python 3.6.9 is currently installed

--- a/detect.py
+++ b/detect.py
@@ -175,7 +175,7 @@ if __name__ == '__main__':
     parser.add_argument('--half', action='store_true', help='use FP16 half-precision inference')
     opt = parser.parse_args()
     print(opt)
-    check_requirements(exclude=('tensorboard', 'pycocotools', 'thop'))
+    check_requirements(exclude=('tensorboard', 'thop'))
 
     if opt.update:  # update all models (to fix SourceChangeWarning)
         for opt.weights in ['yolov5s.pt', 'yolov5m.pt', 'yolov5l.pt', 'yolov5x.pt']:

--- a/hubconf.py
+++ b/hubconf.py
@@ -31,7 +31,7 @@ def _create(name, pretrained=True, channels=3, classes=80, autoshape=True, verbo
     from utils.torch_utils import select_device
 
     check_requirements(requirements=Path(__file__).parent / 'requirements.txt',
-                       exclude=('tensorboard', 'pycocotools', 'thop', 'opencv-python'))
+                       exclude=('tensorboard', 'thop', 'opencv-python'))
     set_logging(verbose=verbose)
 
     fname = Path(name).with_suffix('.pt')  # checkpoint filename

--- a/requirements.txt
+++ b/requirements.txt
@@ -26,5 +26,5 @@ pandas
 
 # extras --------------------------------------
 # Cython  # for pycocotools https://github.com/cocodataset/cocoapi/issues/172
-pycocotools>=2.0  # COCO mAP
+# pycocotools>=2.0  # COCO mAP
 thop  # FLOPs computation

--- a/test.py
+++ b/test.py
@@ -260,6 +260,7 @@ def test(data,
             json.dump(jdict, f)
 
         try:  # https://github.com/cocodataset/cocoapi/blob/master/PythonAPI/pycocoEvalDemo.ipynb
+            check_requirements(['pycocotools'])
             from pycocotools.coco import COCO
             from pycocotools.cocoeval import COCOeval
 
@@ -311,7 +312,7 @@ if __name__ == '__main__':
     opt.save_json |= opt.data.endswith('coco.yaml')
     opt.data = check_file(opt.data)  # check file
     print(opt)
-    check_requirements(exclude=('tensorboard', 'pycocotools', 'thop'))
+    check_requirements(exclude=('tensorboard', 'thop'))
 
     if opt.task in ('train', 'val', 'test'):  # run normally
         test(opt.data,

--- a/train.py
+++ b/train.py
@@ -495,7 +495,7 @@ if __name__ == '__main__':
     set_logging(opt.global_rank)
     if opt.global_rank in [-1, 0]:
         check_git_status()
-        check_requirements(exclude=('pycocotools', 'thop'))
+        check_requirements(exclude=['thop'])
 
     # Resume
     wandb_run = check_wandb_resume(opt)

